train_model.py

```diff
@@ -126,8 +130,9 @@ def message_block(original_atom_state, original_bond_state, connectivity, i):
 
 model = tf.keras.Model(input_tensors, [out])
 
-STEPS_PER_EPOCH = math.ceil(56000 / batch_size)  # number of training examples
-lr = 1E-4
+# STEPS_PER_EPOCH = math.ceil(56802 / batch_size)  # number of training examples
+STEPS_PER_EPOCH = math.ceil(16445 / batch_size)  # number of training examples
```

Comment:
I think the issue was that previously we didn't decay the learning rate fast enough, by increasing steps per epoch we also decreased the learning rate decay. This should decay at a similar rate to the original model?